packages/flutter/lib/src/material/input_border.dart

```diff
@@ -418,12 +418,11 @@ class OutlineInputBorder extends InputBorder {
 
     const double cornerArcSweep = math.pi / 2.0;
     final double tlCornerArcSweep = start < scaledRRect.tlRadiusX
-      ? math.asin((start / scaledRRect.tlRadiusX).clamp(-1.0, 1.0))
+      ? math.acos(1 - start / scaledRRect.tlRadiusX)
```

Comment:
Here's the PR where this line originally came from: https://github.com/flutter/flutter/pull/34515. The new calculation appears to work for that case as well.

Reply:
A suggestion
```dart
final double tlCornerArcSweep = math.asin((1 - start / scaledRRect.tlRadiusX).clamp(0.0, 1.0));
```

Reply:
Thanks, I was worried about not having the clamp in there anymore.  Will try it now.